source/sass/main.scss

```diff
@@ -26,15 +26,32 @@ img {
   max-width: 100%;
 }
 
-ul {
```

Comment:
Not sure, it was in the initial commit I think. Might be there for markdown, though, that's a good guess.